tests/test_interfaces.py

```diff
@@ -8,7 +8,33 @@
 
 import six
 
-from cryptography.utils import InterfaceNotImplemented, verify_interface
+from cryptography.utils import (
+    InterfaceNotImplemented, register_interface_if, verify_interface
+)
+
+
+def test_register_interface_if_true():
+    @six.add_metaclass(abc.ABCMeta)
+    class SimpleInterface(object):
+        pass
+
+    @register_interface_if(1 == 1, SimpleInterface)
+    class SimpleClass(object):
+        pass
+
+    assert issubclass(SimpleClass, SimpleInterface) is True
```

Comment:
Why issubclass and not isinstance here? It probably doesn't matter but I'm curious if there's any effective difference for our purposes